### PR TITLE
[RT] Updates the readme for when we finally make RT the main branch

### PR DESCRIPTION
**Pull request by ike709**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,18 @@
 
 This project is currently in early stages, and is not feature-complete.
 
-The compiler and server should work fine for Linux-based machines, but we currently rely on WPF and WebView2 for the client, so it's Windows-only.
+The compiler and server should work fine for Linux-based machines. The client should also work fine on Linux but none of the developers use it. More details and potentially useful troubleshooting for running on Linux can be found in the haphazardly written FAQ [here](https://github.com/wixoaGit/OpenDream/blob/RobustToolbox/LINUX_FAQ.md).
```
Comment (ZeWaka) [inline README.md:7]: lolololol

**Review by ZeWaka — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (wixoaGit) [inline README.md:7]: The LINUX_FAQ.md only contains outdated information about CEF and setup. I'm removing it.

This line in the readme was meant to point out that one piece of the software was incompatible with Linux, which is no longer the case. Removing it entirely seems best.

**Review by wixoaGit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ike709) [inline README.md:7]:
```suggestion
All parts of OpenDream should work fine on Windows and Linux, though the latter is not used as much and therefore isn't as thoroughly tested.
```

**Comment by ike709** [comment]
Tweaked the linux mention to just say we don't test it as much, which is accurate.

**Review by wixoaGit — APPROVED** [review]

**Pull request merged**